tasks/installation.yml

```
@@ -1,6 +1,11 @@
 ---
 
 - name: install motd related packages
-  package:
+  ansible.builtin.package:
     name: '{{ motd_packages }}'
     state: present
+
+- name: install cowsay packages
+  ansible.builtin.package:
+    name: 'cowsay'
+    state: present
```

i'm here for cowsay, but what if i already have a non package based installation of cowsay that supports offensive mode?

Maybe installing cowsay needs to be skippable?
```suggestion
- name: install cowsay packages
  ansible.builtin.package:
    name: 'cowsay'
    state: present
  when: motd_cowsay is true
```

---

Resolved in [401f973](https://github.com/adfinis/ansible-role-motd/pull/5/commits/401f97303d93b763483a0ace109d64833015fa95)